Task1/Node/fs-demo.js

```diff
@@ -0,0 +1,45 @@
+const ghash = require('./hash-generation.js');
+
+const fs = require("fs");
+const path = fs.readdirSync("./utils");
+//TODO check if utils is path or directory ?
+var stats = fs.statSync("./utils");
+console.log('Is utlis a file or directory ?');
+console.log('is file ? ' + stats.isFile());
+console.log('is directory ? ' + stats.isDirectory());
+
+//TODO
+//  read all file names in the directory
+// if you find a directory inside a path you go read recursively
+
+console.log('All files and Directory names listed : ');
+async function print(path) {
```

Comment:
its a great start with async functions.. however you need to name it better.
give it a relevant name.

Reply:
rename the function as - 'printDirectoryAndFileNames' and modified the code accordingly